src/main/java/school/hei/tsinjo/repository/mapper/JPaymentMapper.java

```diff
@@ -12,26 +10,33 @@
 @Component
 public class JPaymentMapper {
   public Payment toDomain(JPayment jPayment) {
-    return switch (PspType.values()[0]) {
```

Comment:
Write bakc my code! This will statically enforce any addition of new PSP will cause the compiler to tell you where are all the places you should bring modifications!